### PR TITLE
Fix dependencies, mainly eslint

### PR DESCRIPTION
**Pull request by draperunner**
Wrong version of `eslint-plugin-import` led to an error in the rule `import/extensions` from the airbnb config. 
Downgrade redux-persist to fix npm "wrong dependency" warning.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,13 +64,13 @@
...
-    "eslint-import-resolver-webpack": "0.6.0",
-    "eslint-plugin-flowtype": "^2.19.0",
-    "eslint-plugin-import": "2.0.0",
-    "eslint-plugin-jsx-a11y": "2.2.2",
-    "eslint-plugin-react": "6.3.0",
+    "eslint": "^3.8.1",
+    "eslint-config-airbnb": "^12.0.0",
+    "eslint-import-resolver-webpack": "^0.6.0",
+    "eslint-plugin-flowtype": "^2.25.0",
+    "eslint-plugin-import": "^1.16.0",
```
Comment (ekmartin) [inline package.json]: I think the lastest version of eslint-config-airbnb uses eslint-plugin-import v2?

Comment (ekmartin) [inline package.json]: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json#L66

Comment (ekmartin) [inline package.json]: If you run the command [here](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1) it should set the correct peer versions.

Comment (draperunner) [inline package.json]: I tried that a couple of times, but it didn't seem to fix the problem with the `import/extensions` rule...

Comment (draperunner) [inline package.json]: The script actually returns the dependencies that are in this PR.

Comment (ekmartin) [inline package.json]: Ah! Looks like they've only released it in eslint-config-airbnb-**base** so far. Sorry :)

**Comment by larseen** [comment]
@draperunner Whats status here?

**Comment by draperunner** [comment]
This PR is probably outdated. I'll check on it tonight.

**Comment by draperunner** [comment]
Outdated. Closing.

**Pull request closed without merging**